addons/web_editor/static/src/less/web_editor.ui.less

```diff
@@ -284,7 +284,7 @@ html[lang] > body.editor_enable [data-oe-translation-state] {
 
 // MANIPULATORS
 #oe_manipulators {
-    @odoo-handle-edge-size: 8px;
+    @odoo-handle-edge-size: 1px;
```

Comment:
I like the visual for margin and col-offset, just two comments:
- I am used to edit them by dragging the margin line (not the margin button) and now when there is no margin, this is difficult again as it is 1px size
- Not sure I like that the margins are blinking every time we select a snippet